src/test/java/org/prebid/server/bidder/colossus/ColossusBidderTest.java

```diff
@@ -91,19 +93,20 @@ public void makeHttpRequestsShouldMakeOneRequestPerImp() {
                 requestBuilder -> requestBuilder.imp(Arrays.asList(
                         givenImp(identity()),
                         Imp.builder()
-                                .ext(mapper.valueToTree(ExtPrebid.of(null, ExtImpColossus.of("otherTagId"))))
+                                .ext(mapper.valueToTree(
```

Comment:
I was talking about something like this. Use this mechanism, please.
`givenImp(impBuilder -> impBuilder.ext(mapper.valueToTree(`
`ExtPrebid.of(null, ExtImpColossus.of("otherTagId", null))))))));`